samples/crypto/ecdh/README.rst

```diff
@@ -7,31 +7,28 @@ Crypto: ECDH
    :local:
    :depth: 2
 
+The ECDH sample shows how to perform an Elliptic-curve Diffie–Hellman key exchange to allow two parties to obtain a common secret.
+
 Overview
 ********
 
-The ECDH sample shows how to perform an Elliptic-curve Diffie–Hellman key exhange to allow two parties to obtain a common secret.
+The sample follows these steps to perform an ECDH key exchange:
+
+1. First, the sample performs initialization:
 
-The sample peforms an ECDH exchange:
-   #. The Platform Security Architecture (PSA) API is initialized.
-   #. Two ECDH keypairs are generated and imported to the PSA keystore.
-   #. ECDH key echange is performed.
-   #. The keypairs are removed from the PSA keystore.
+   a. The Platform Security Architecture (PSA) API is initialized.
+   #. Two random ECDH key pairs are generated and imported into the PSA crypto keystore.
```

Comment:
ECDH key pairs needs to be changed ECC key pairs. There is nothing that makes these key pairs specific to an ECDH exchange.

Bonus points: Explain ECC  as "Elliptic Curve Cryptography" as this is possibly the first time that term is used (and the acronym is probably not that well known). PSA uses this acronym for their APIs as well...

@Vge0rge : This probably needs to be fixed in the code if the term ECDH keypair is used

Reply:
I used the term ECDH because the algorithm that we set in the psa_apis is ECDH. But I understand that if we want to be technically correct probably this needs to be changed in the code as well. The function names comments as well. So my opinion is to leave it like that for now and just create a PR after the release to fix both the code and documentation.

Reply:
That is acceptable for me, @ioannisg?

Reply:
Changed to ECC as suggested